lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -78,9 +80,9 @@ class RenderForm extends React.Component {
 
       request.onreadystatechange = function success() {
         if (request.readyState === 4 && request.status === 201) {
-          // TODO needs to remove after implementing success page
-          const alert = 'Request submitted successfully.';
-          alert();
+          $this.setState({
```

Comment:
Using the [bind function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) will help access the proper "this" instead of reassigning it to `$this`.